test/Test/BaseDAO/Proposal.hs

```diff
@@ -7,13 +7,14 @@ module Test.BaseDAO.Proposal
 
 import Universum hiding (compare, drop, (>>))
 
-import Lorentz
+import Lorentz hiding (map)
```

Comment:
An alternative solution is to use `fmap` :sweat_smile:

Reply:
Oh right, that's simpler.